templates/pg_dump.sh.epp

```diff
@@ -1,3 +1,20 @@
+<%- |
+  Boolean                                  $compress,
+  Array                                    $databases,
+  Optional[String[1]]                      $db_user,
+  Boolean                                  $delete_before_dump,
+  String[1]                                $dir,
+  Enum['plain','custom','directory','tar'] $format,
+  Array                                    $optional_args,
```

Comment:
Is this `Array[String[1]]` or do we also expect `Integer` in there?

Reply:
I took over the current types in #1397 or these which are currently in the module. If there are adjustments over all, I would do these later.